src/js/shield_defs.js

```diff
@@ -500,7 +500,7 @@ export function loadShields() {
       textColor: Color.shields.brown,
       colorLighten: Color.shields.brown,
     },
-    ["HIST"],
+    ["XYZA"],
```

Comment:
This change will be reverted prior to merging. I changed it in order to demonstrate what a before-and-after sample would look like in the build actions.